api/src/main/java/org/apache/cloudstack/api/command/user/resource/UpdateResourceCountCmd.java

```diff
@@ -117,15 +124,14 @@ public long getEntityOwnerId() {
     @Override
     public void execute() {
         List<? extends ResourceCount> result =
-                _resourceLimitService.recalculateResourceCount(_accountService.finalyzeAccountId(accountName, domainId, projectId, true), getDomainId(), getResourceType());
+                _resourceLimitService.recalculateResourceCount(_accountService.finalyzeAccountId(accountName, domainId, projectId, true), getDomainId(), getResourceType(), getTag());
 
         if ((result != null) && (result.size() > 0)) {
             ListResponse<ResourceCountResponse> response = new ListResponse<ResourceCountResponse>();
             List<ResourceCountResponse> countResponses = new ArrayList<ResourceCountResponse>();
 
             for (ResourceCount count : result) {
                 ResourceCountResponse resourceCountResponse = _responseGenerator.createResourceCountResponse(count);
-                resourceCountResponse.setObjectName("resourcecount");
```

Comment:
Would this change the json key in the output?

Reply:
@vishesh92 removed this as ApiResponseHelper already sets the same thing in `createResourceCountResponse`